Multisig/UI/Settings/Passcode/PasscodeViewController.swift

```diff
@@ -225,7 +257,18 @@ class EnterPasscodeViewController: PasscodeViewController {
     }
 }
 
-class ChangePasscodeViewController: PasscodeViewController {
+class ChangePasscodeViewController: EnterPasscodeViewController {
```

Comment:
instead of subclassing it is better to have a variable that is configured at the call site - for the title and for the tracking event. This way we don't have too many classes (it becomes confusing)

Reply:
changed